tests/conftest.py

```diff
@@ -1,5 +1,10 @@
-from pytest_reqs import check_requirements
+# Allow import to fail to avoid annoying developers
+try:
+    from pytest_reqs import check_requirements
+except ImportError:
```

Comment:
Should we have this indicated in the logger that this test isn't being run and the dev would need to install pytest-reqs plugin ?

Reply:
Sounds like a good idea,.
https://gitlab.com/coala/mobans/issues/74